src/ctim/schemas/note.cljc

```diff
@@ -20,6 +21,8 @@ For example, an analyst may add a Note to a Campaign object created by another o
    (f/entry :entity_id c/Reference)
    (f/entry :content c/Markdown))
   (f/optional-entries
+   (f/entry :note_class v/NoteClass)
+   (f/entry :related_entities (f/seq-of c/Reference))
```

Comment:
I would use `note_type` in order to be consistent with `relationship_type` of [relationship schema](https://github.com/threatgrid/ctim/blob/4aa3a14db97715c493a05dec581984b2ae542af5/src/ctim/schemas/relationship.cljc#L18).
I would also use `entity_ids` like `entity_id` of [feedback](https://github.com/threatgrid/ctim/blob/4aa3a14db97715c493a05dec581984b2ae542af5/src/ctim/schemas/feedback.cljc#L17) and `asset_id` of asset-mappings and asset-properties.

Reply:
I can't use NoteTypenas it conflicts with another eponym field in the vocabularies

Reply:
I debates the use of entity_ids but I wanted perhaps to convey the notion of a primary target for the relation

Reply:
we finally decided to switch to `related_entities` as a single vector of maps including the target type for convenience.